### DISCUSSION TITLE
New Testing Framework

### DISCUSSION THREAD

**Discussion by mikewolfd** [FearlessSolutions/engineering-practice-domains#19] [Show and tell]
I just saw [this](https://github.com/kolodny/safetest) a few days ago and thought it was really interesting, it pretty much combines E2E and Integration/Unit testing in a cohesive and effective way. It's integrated Playwright directly with jest tests and creates a unified way of writing tests that checks both selectively I guess?

Thought of you @cyber-decker

**Comment by cyber-decker** [comment]
Yesssssss. I looked at this a few months back and it was really interesting.  Very VERY fresh and new.  I've been meaning to grab it and give it a try when I have some spare time, but that is at a premium right now.  Probably worth digging in to this very soon and getting ahead of the curve.
